src/main/java/fish/payara/extras/diagnostics/collection/CollectorService.java

```diff
@@ -165,32 +181,16 @@ public int executeCollection() {
      * @param zipOutputStream
```

Comment:
Adjust the javadoc please - you've edited the parameters

Reply:
resolved in #21